seqr/utils/elasticsearch/constants.py

```diff
@@ -246,16 +248,16 @@
 CORE_FIELDS_CONFIG = {
     'alt': {},
     'contig': {'response_key': 'chrom'},
-    'end': {'format_value': long},
+    'end': {'format_value': int},
```

Comment:
this will absolutely break important seqr code. The values for variant start and end coordinates are larger than ints (at least in python 2). You need a different fix for this, although again I don't understand why this file is a blocker for the dataset tests?

Reply:
I will add `from builtins import int` which will turn all `int` to be same as `long`.

Reply:
The file is imported by the `seqr/utils/elasticsearch/utils.py` and `seqr/views/utils/data_utils.py`.